Application/Manual/Support/Frontend.php

```diff
@@ -86,4 +89,54 @@ public function formTicket()
             )
         );
     }
+
+    /**
+     * @param null $Ticket
+     *
+     * @return Stage
+     */
+    public function frontendRequest($Ticket = null)
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$Ticket" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)